k2hb_userdata.tpl

```diff
@@ -20,6 +20,17 @@ export HTTPS_PROXY="$https_proxy"
 export no_proxy="${non_proxied_endpoints}"
 export NO_PROXY="$no_proxy"
 
+# Support for HCS Image (Single Vol)
+echo "Disable iptables (AWS security group rules take precedence)"
+iptables -P INPUT ACCEPT
+iptables -P OUTPUT ACCEPT
+iptables -P FORWARD ACCEPT
+iptables -F
+
+#Precautionary extension of log folder based on htme config
+lvextend -L10G /dev/rootvg/varlogvol
```

Comment:
We may want to use up any additional disk on root or some other LVM that is important:

`lvextend -l 100%FREE /dev/rootvg/rootvol`
and 
`xfs_growfs /dev/mapper/rootvg-rootvol`

This way we don't have disk space that is unused.

Reply:
It looks like /var/log is hungry for more disk in prod. Might be worth extending that to 50G as well as using any additional disk for root